src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/ShowUniqueValuesInColumn.java

```diff
@@ -22,12 +22,13 @@ public static void main(String[] args) {
 
         pipeline.setInput(pipelineInput);
 
-        Dataset dataset = new MemoryDataset(pipeline);
+        Dataset dataset = new MemoryDataset(pipeline).setCollectUniqueValues(true); //enables calculating unique values within a dataset
         dataset.load().waitForRecordsToLoad();
+        dataset.load().waitForColumnStatsToLoad(); //loads all columns
```

Comment:
@sah2020 Let's leave the "comments" to the CMS if needed. 

BTW, our goal in the library is to replace the need for comments with methods and good naming, except in very rare cases.

Reply:
ok got it